tests/src/Loader/classloader/generics/Instantiation/Nesting/NestedGenericClasses.csproj

```diff
@@ -12,6 +12,9 @@
     <OutputType>Exe</OutputType>
     <CLRTestKind>BuildAndRun</CLRTestKind>
     <CLRTestPriority>1</CLRTestPriority>
+
+    <!-- Test fails to build on .Net Core 3.0 preview3 -->
+    <DisableProjectBuild Condition="'$(__BuildOS)' == 'OSX'">true</DisableProjectBuild>
```

Comment:
Just a note, these look to also fail on Windows if using the desktop MSBuild.

Reply:
We do not have an exclude mechanism for desktop. So I believe this change is ready to merge.

Reply:
@jashook, you should just be able to test `'$(RunningOnCore)' != 'true'`, which is how you determine if you are running on the Desktop or .NET Core MSBuild (and therefore, which compiler task you will be invoking).